be/src/service/backend_service.cpp

```diff
@@ -339,4 +339,7 @@ void BackendService::get_stream_load_record(TStreamLoadRecordResult& result,
     }
 }
 
+void BackendService::clean_trash() {
+    StorageEngine::instance()->start_trash_sweep(nullptr, true);
```

Comment:
It may takes a very long time to clean the trash. So I suggest to use a async call.

Reply:
> 
> 
> It may takes a very long time to clean the trash. So I suggest to use a async call.
I think this is already async, because of I use `oneway` to define the function at thrift file. 
[`gensrc/thrift/BackendService.thrift`](https://github.com/apache/incubator-doris/blob/3c899f690d124d612b0d91949872cd9fb9faab80/gensrc/thrift/BackendService.thrift)
`oneway void clean_trash();`